content/blog/2021/23/camelDebuggerForIntelliJ/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Camel Debugger For IntelliJ"
+date: 2021-12-23T07:53:49-07:00
+authors: ["javaduke"]
+categories: ["Tooling"]
+preview: "Apache Camel plugin for IntelliJ now features a debugger for XML routes"
+---
+
+The recent release of the [Apache Camel plugin for IntelliJ version v0.8.0](https://plugins.jetbrains.com/plugin/9371-apache-camel) includes the first tech preview of the Camel Route Debugger. The debugger is currently available on Maven-based Camel projects and routes defined in the XML DSL. The minimum recommended Camel version is 3.15.0-SNAPSHOT (older versions also may work, but the functionality is limited).
+
+# Features
+
+The first tech preview includes the following features:
+
+- breakpoints inside Camel routes in XML DSL;
+- conditional breakpoints with Simple language predicates;
```

Comment:
optional: for consistency, I think you should provide capital letter to start the bullet